pkg/client/invokeandexecute.go

```diff
@@ -27,9 +27,13 @@ func invokeAndExecute[Caveats, Out any](
 	with ucan.Resource,
 	caveats Caveats,
 	successType schema.Type,
-	options ...delegation.Option,
```

Comment:
aren't there any other options we could ever be interested in passing here?

Reply:
Yeah, that seems fair. `invokeAndExecute` would take them, but not the exported client methods (which themselves might decide to create and pass some to `invokeAndExecute`. I'll add it back to the signature here.